upstream/CVE-2023-2431.json

```
@@ -0,0 +1,113 @@
+{
+  "id": "CVE-2023-2431",
+  "modified": "2023-06-15T14:42:32Z",
+  "published": "2023-06-15T14:42:32Z",
+  "summary": "Bypass of seccomp profile enforcement ",
+  "details": "A security issue was discovered in Kubelet that allows pods to bypass the seccomp profile enforcement. Pods that use localhost type for seccomp profile but specify an empty profile field, are affected by this issue. In this scenario, this vulnerability allows the pod to run in unconfined (seccomp disabled) mode. This bug affects Kubelet.",
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/kubelet"
+      },
+      "severity": [
+        {
+          "type": "CVSS_V3",
+          "score": "CVSS:3.1/AV:L/AC:L/PR:H/UI:N/S:U/C:L/I:L/A:N"
+        }
+      ],
+      "ranges": [
+        {
+          "events": [
+            {
+              "introduced": "0"
+            },
+            {
+              "fixed": "1.24.14"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/kubelet"
+      },
+      "severity": [
+        {
+          "type": "CVSS_V3",
+          "score": "CVSS:3.1/AV:L/AC:L/PR:H/UI:N/S:U/C:L/I:L/A:N"
+        }
+      ],
+      "ranges": [
+        {
+          "events": [
+            {
+              "introduced": "1.25.0"
+            },
+            {
+              "fixed": "1.25.9"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/kubelet"
+      },
+      "severity": [
+        {
+          "type": "CVSS_V3",
+          "score": "CVSS:3.1/AV:L/AC:L/PR:H/UI:N/S:U/C:L/I:L/A:N"
+        }
+      ],
+      "ranges": [
+        {
+          "events": [
+            {
+              "introduced": "1.26.0"
+            },
+            {
+              "fixed": "1.26.4"
```

```suggestion
              "fixed": "1.26.5"
```
[issue](https://github.com/kubernetes/kubernetes/issues/118690):
> Fixed Versions
v1.27.2
v1.26.5

```
➜  kubernetes git:(master) git tag --contains 67c667c788a1227842d8626f052c847f51886f85 | sort -V | head -n 5
v1.26.5
v1.26.6
v1.26.7
v1.26.8
v1.26.9
```

---

fixed in https://github.com/aquasecurity/vuln-list-k8s/pull/91/commits/359066782adb6f0c293cf2b3730a07421785369b